### PR TITLE
Improve root README

### PR DESCRIPTION
**Pull request by armando-rodriguez-cko**
Improve root README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+[![build-status](https://github.com/checkout/sdk-samples/actions/workflows/create_go_package.yml/badge.svg)](https://github.com/checkout/sdk-samples/actions/workflows/create_go_package.yml)
+
+[![build-status](https://github.com/checkout/sdk-samples/actions/workflows/create_node_package.yml/badge.svg)](https://github.com/checkout/sdk-samples/actions/workflows/create_node_package.yml)
+
+[![GitHub license](https://img.shields.io/github/license/checkout/sdk-samples.svg)](https://github.com/checkout/sdk-samples/blob/main/LICENSE)
+
+In this repository you can get a sample of integration in the different languages that the SDK Team of Checkout give support.
+
+## SDK language support
+SDK Checkout support these languages:
```
Comment (martinseco) [inline README.md:22]: supports

File: README.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+- <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/java/java-original.svg" width="20" height="20"> JAVA
+- <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/dot-net/dot-net-original.svg" width="20" height="20"> NET
+- <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/php/php-original.svg" width="20" height="20"> PHP
+- <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/python/python-original.svg" width="20" height="20"> Python
+- <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/ruby/ruby-original.svg" width="20" height="20"> Ruby
+- <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/go/go-original.svg" width="20" height="20"> Go
+- <img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/javascript/javascript-original.svg" width="20" height="20"> Javascript (NodeJs)
+
+## Sample Repositories
+Next, you can access to the repositories here and check the documentation:
```
Comment (martinseco) [inline README.md:33]: Remove the "Next,"

**Review by martinseco — APPROVED** [review]
Looks good. Just a few changes

File: README.md (hunk 1 above)
Comment (armando-rodriguez-cko) [reply README.md:22]: 👍🏼

File: README.md [hunk 3]
```diff
@@ -1 +1,67 @@
...
+- [<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/dot-net/dot-net-original.svg" width="20" height="20"> NET](https://github.com/checkout/sdk-samples/tree/main/net)
+- [<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/php/php-original.svg" width="20" height="20"> PHP](https://github.com/checkout/sdk-samples/tree/main/php)
+- [<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/python/python-original.svg" width="20" height="20"> Python](https://github.com/checkout/sdk-samples/tree/main/python)
+- [<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/ruby/ruby-original.svg" width="20" height="20"> Ruby](https://github.com/checkout/sdk-samples/tree/main/ruby)
+- [<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/go/go-original.svg" width="20" height="20"> Go](https://github.com/checkout/sdk-samples/tree/main/go)
+- [<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/javascript/javascript-original.svg" width="20" height="20"> Javascript (NodeJs)](https://github.com/checkout/sdk-samples/tree/main/node)
+
+## Sample Packages
+Also, you can access directly to the packages here:
```
[the remark this answers is not in the document]
Comment (armando-rodriguez-cko) [reply README.md:43]: 🥇

**Pull request merged**